client/blocks/comments/comment-approve-action.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import classnames from 'classnames';
+
+const CommentApproveAction = ( { translate, status, approveComment, unapproveComment } ) => {
+	const isApproved = status === 'approved';
+	const buttonStyle = classnames( 'comments__comment-actions-approve', {
+		'is-approved': isApproved
+	} );
+
+	return (
+		<button className={ buttonStyle } onClick={ ! isApproved ? approveComment : unapproveComment }>
+			<Gridicon icon="checkmark" size={ 18 } />
+			<span className="comments__comment-actions-like-label">{ isApproved ? translate( 'Approved' ) : translate( 'Approve' ) }</span>
```

Comment:
:disappointed: 31 existing translations will be lost with this change.
If the string must change, the following option has already been translated **28** times:
`translate( 'Approved', { context: 'verb: past-tense'} )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).